cmd/minishift/cmd/root.go

```diff
@@ -59,6 +60,10 @@ const (
 	invalidProfileName    = "Profile names must consist of alphanumeric characters only."
 )
 
+var preventHomeDirCreationForCmds = []string{
```

Comment:
abbrev. ?

Reply:
Actually these commands skip persistant prerun. Home directory creation is a by product of execution of persistent prerun.

Reply:
So I suggestion would be `noPersistentPreRunCmds` or something similar.

Reply:
@gbraad what do you think about `noPersistentPreRunForCmds`? or any similar name?

Reply:
I am fine with this, as the meaning is more important, and skipping prerun is what happens.

Reply:
but it might be helpful to ayt least explain what that means...
... or be clear in the rest of the preRun what each step does.